pkg/webservice/handlers.go

```diff
@@ -457,7 +457,36 @@ func getClusterConfig(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func updateConfig(w http.ResponseWriter, r *http.Request) {
+func createClusterConfig(w http.ResponseWriter, r *http.Request) {
+	writeHeaders(w)
+
+	queryParams := r.URL.Query()
+	_, dryRunExists := queryParams["dry_run"]
+	if len(queryParams) == 0 || !dryRunExists {
+		http.Error(w, "Mandatory parameters are missing in URL path. Please check the usage documentation", http.StatusBadRequest)
+		return
+	}
+	if queryParams.Get("dry_run") != "1" {
+		http.Error(w, "Invalid query params. Please check the usage documentation", http.StatusBadRequest)
+		return
+	}
```

Comment:
This seems to require that dry run is set, which is confusing. Could the error message maybe state what parameter is missing or what parameter is invalid?

Reply:
Right now we support only dry_run=1. I think we should document it and also include it in the error message.

Reply:
Taken care. Filed a separate JIRA - https://issues.apache.org/jira/browse/YUNIKORN-515 to track doc/user guide changes.